docs/ref/databases.txt

```diff
@@ -103,7 +103,7 @@ below for information on how to set up your database correctly.
 PostgreSQL notes
 ================
 
-Django supports PostgreSQL 9.6 and higher. `psycopg2`_ 2.5.4 or higher is
+Django supports PostgreSQL 10 and higher. `psycopg2`_ 2.5.4 or higher is
```

Comment:
We probably ought to check this version of `psycopg2` as it is very old. With supporting Python 3.8 as a minimum in 4.0 this is probably not right.

Obviously separate PR - just putting it on the radar. I was giving some thought to minimum versions in ticket-28905, #10663 - probably something to pick up there.

Reply:
Normally we don't bump drivers versions without a specific reason, e.g. `2.5.4` added support for `JSONB`. If you think it's worth bumping or support is already broken, please create a ticket with rationale.